dockerfiles/centos/is/Dockerfile

```diff
@@ -31,7 +31,7 @@ ARG USER_HOME=/home/${USER}
 ARG JAVA_HOME=${USER_HOME}/java
```

Comment:
@ThilinaManamgoda A new CentOS based Docker image with AdoptOpenJDK is available. Shall we please use it here? It would be better to use this since, generally it takes a longer time to build the CentOS based images since, we download the AdoptOpenJDK distribution directly from GitHub.

For a sample, please see the WSO2 API Manager version `3.1.0` changes from (https://github.com/wso2/docker-apim/pull/289/files#diff-3d8f51ac2e166a345797d6469b8de8eb).

Reply:
updated